plugins/hls-change-type-signature-plugin/test/testdata/TErrorGivenPartialSignature.hs

```diff
@@ -0,0 +1,4 @@
+module ErrorGivenPartialSignature where
```

Comment:
missing expected file?

Reply:
No this one shouldn't generate any CodeAction as the error doesn't provide enough information